### ISSUE TITLE
Resources

### ISSUE THREAD

**Issue by nirlipo** [AI-Planning/planning-as-a-service#1]
Intro about Flask+Celery+redis:
  - https://ljvmiranda921.github.io/notebook/2019/11/08/flask-redis-celery-mcdo/

Tutorial On Flask:
  - https://code.visualstudio.com/docs/python/tutorial-flask

Tutorial Dockerizing:
  - https://code.visualstudio.com/docs/containers/quickstart-python

Tutorial on adding debugger to Flask:
  - https://blog.theodo.com/2020/05/debug-flask-vscode/

Code:
  - https://github.com/microsoft/python-sample-vscode-flask-tutorial/tree/tutorial
  - https://stackabuse.com/asynchronous-tasks-using-flask-redis-and-celery/

Flask cannot run with python3.8. Unresolved issue:
  - https://github.com/pallets/werkzeug/issues/1551

Need to use python3.6 with the current requirements.txt. We need to find out if this issue can be solved, If find it surprising that FLASK wouldn't work with python3.8

**Comment by haz** [comment]
Hrmz...isn't that issue resolved?

**Comment by nirlipo** [comment]
I meant this post as a way to track useful resources and insights at this early stage

**Comment by haz** [comment]
Sorry, didn't mean issue #1. I meant the werkzeug issue: https://github.com/pallets/werkzeug/issues/1551

**Comment by nirlipo** [comment]
Good point, now we are working with latest Flask version and python 3.8.
